content/zh-cn/docs/tasks/run-application/horizontal-pod-autoscale.md

```diff
@@ -93,7 +92,7 @@ horizontal pod autoscaling.
 {{< mermaid >}}
 graph BT
 
-hpa[Pod 水平自动扩缩] --> scale[规模]
+hpa[HorizontalPodAutoscaler] --> scale[规模]
```

Comment:
```suggestion
HPA[HorizontalPodAutoscaler] --> scale[规模]
```

Reply:
Seems we should change the english first?
Not sure whether the mermaid works properly after replacing `hpa` with `HPA`.
<img width="553" height="409" alt="image" src="https://github.com/user-attachments/assets/64032342-44ea-4ed8-8530-63641650f1e1" />
These variables are not a user-facing word. Readers can only view those text in `[]`.

Reply:
Okay. Just checked the English version. This should be fine.